std/internal/cstring.d

```diff
@@ -234,31 +233,30 @@ private:
     static TempCStringBuffer trustedVoidInit() { TempCStringBuffer res = void; return res; }
 }
 
-private To[] trustedRealloc(To)(To[] buf, size_t i, To[] res, size_t strLength, bool res_is_onstack)
+private To[] trustedRealloc(To)(To[] buf, size_t strLength, bool buf_is_onstack)
```

Comment:
If you already change this, I think we should stick to the [DStyle](https://dlang.org/dstyle.html), e.g. sth. like `bufIsOnStack`

Reply:
Will do when at home.